modules/installation-and-upgrade/pages/container-deployment/suma/proxy-deployment-vm-suma.adoc

```diff
@@ -70,6 +74,8 @@ This table shows the hardware requirements for deploying {productname} Proxy.
 |===
 
 
+// FIXME Do we need the following note here?
+
 [NOTE]
 ====
 [path]``/var/lib/containers/storage/volumes/srv-www``
```

Comment:
As we already established at [proxy-deployment-uyuni.adoc?plain=1#L65](https://github.com/uyuni-project/uyuni-docs/blob/8601d3295eb22545f51569e904b7e9480227cf9b/modules/installation-and-upgrade/pages/container-deployment/uyuni/proxy-deployment-uyuni.adoc?plain=1#L65) this is wrong and should be changed.